internal/pkg/template/templates/workloads/partials/cf/autoscaling.yml

```
@@ -113,8 +113,21 @@ AutoScalingPolicyECSServiceAverageCPUUtilization:
     TargetTrackingScalingPolicyConfiguration:
       PredefinedMetricSpecification:
         PredefinedMetricType: ECSServiceAverageCPUUtilization
+      {{if .Autoscaling.CPUCooldown}}
+      {{- if .Autoscaling.CPUCooldown.ScaleInCooldown}}
+      ScaleInCooldown: {{.Autoscaling.CPUCooldown.ScaleInCooldown}}
+      {{- else}}
       ScaleInCooldown: 120
+      {{- end}}
+      {{- if .Autoscaling.CPUCooldown.ScaleOutCooldown}}
+      ScaleOutCooldown: {{.Autoscaling.CPUCooldown.ScaleOutCooldown}}
+      {{- else}}
       ScaleOutCooldown: 60
+      {{- end}}
+      {{else}}
+      ScaleInCooldown: 120
+      ScaleOutCooldown: 60
+      {{end}}
```

could be simplified: 
```suggestion
      {{- if .Autoscaling.CPUCooldown.ScaleInCooldown}}
      ScaleInCooldown: {{.Autoscaling.CPUCooldown.ScaleInCooldown}}
      {{- else}}
      ScaleInCooldown: 120
      {{- end}}
      {{- if .Autoscaling.CPUCooldown.ScaleOutCooldown}}
      ScaleOutCooldown: {{.Autoscaling.CPUCooldown.ScaleOutCooldown}}
      {{- else}}
      ScaleOutCooldown: 60
      {{- end}}
```

---

Ah thanks! I missed that this could be simplified when making cooldowns a non-pointer value in AutoscalingOpts.